app/src/main/java/com/hedvig/app/feature/marketing/marketpicked/MarkedPicked.kt

```diff
@@ -0,0 +1,98 @@
+package com.hedvig.app.feature.marketing.marketpicked
```

Comment:
The file name probably was meant to be MarkerPicked instead of MarkedPicked right :D